File: conformance/bundles/18-emphasis-parameter/mention.steps.cs
// C# sibling of mention.steps.ts (bundle 18-emphasis-parameter). {emph} is a
// built-in parameter type (Markdown emphasis); matching is what conformance pins.
using Varar;
using Varar.Core;

namespace Varar.Corpus.B18;

public static class MentionSteps
{
    public static void Register(Steps s)
    {
        s.Stimulus("I mention {emph}", (state, text) => null);
    }

    public static Value State() => Value.Null;
}

File: conformance/bundles/18-emphasis-parameter/mention.steps.rs
//! Rust sibling of `mention.steps.ts` (bundle `18-emphasis-parameter`).
//!
//! `{emph}` is a built-in parameter type: Markdown emphasis, with only the inner
//! text passed to the handler. Matching is what conformance pins here.

use varar::Steps;

pub fn register(s: &mut Steps<()>) {
    s.stimulus("I mention {emph}", |ctx: (), _who: String| Ok(ctx));
}

pub fn state() {}
